tests/end2end/features/hints.feature

```diff
@@ -81,6 +81,20 @@ Feature: Using hints
         And I hint with args "all userscript (testdata)/userscripts/echo_hint_text" and follow a
         Then the message "Follow me!" should be shown
 
+    Scenario: Yanking to clipboard
+        When selection is supported
```

Comment:
This checks whether primary selection is supported - however this test (using the normal clipboard) can run without a primary selection (e.g. on Windows) too.